flyteadmin/pkg/workflowengine/impl/compiler.go

```diff
@@ -37,7 +37,7 @@ func (c *workflowCompiler) CompileWorkflow(
 
 	compiledWorkflowClosure, err := compiler.CompileWorkflow(primaryWf, subworkflows, tasks, launchPlans)
 	if err != nil {
-		return nil, errors.NewFlyteAdminErrorf(codes.InvalidArgument, "failed to compile workflow with err %v", err)
+		return nil, errors.NewFlyteAdminError(codes.InvalidArgument, err.Error())
```

Comment:
To remove duplicate errors.

![Screenshot 2024-08-08 at 3 19 53 PM](https://github.com/user-attachments/assets/dd421b11-078a-4027-bd32-93f20109bffc)